more_itertools/more.py

```diff
@@ -12,11 +12,32 @@
 from .recipes import take
 
 __all__ = [
-    'bucket', 'chunked', 'collapse', 'collate', 'consumer',
-    'distinct_permutations', 'distribute', 'first', 'ilen',
-    'interleave_longest', 'interleave', 'intersperse', 'iterate', 'one',
-    'padded', 'peekable', 'side_effect', 'sliced', 'split_after',
-    'split_before', 'spy', 'unique_to_each', 'windowed', 'with_iter',
+    'bucket',
```

Comment:
Some hate this style; others get tired of re-formatting things... :parasol_on_ground: